src/cli.py

```diff
@@ -20,6 +23,19 @@ def parse_parameter_string(param_str: str) -> dict:
     return dict_to_return
 
 
+def get_root_path_from_env_var(env_var_name: str) -> Path:
+    os_default_root_path_mapping = {
+        "Windows": "c://",
+        "Darwin": "/",
+        "Linux": "/",
+        "Java": "/",
+        "": "/",
+    }
+    return Path(
+        os.getenv(env_var_name, os_default_root_path_mapping[platform.system()])
```

Comment:
could default the dict call here for safety:
```suggestion
        os.getenv(env_var_name, os_default_root_path_mapping.get(platform.system(), "/"))
```